src/features/moreTab/more/MoreScreen.tsx

```diff
@@ -128,6 +129,13 @@ const MoreScreen = () => {
     )
   }, [t, dispatch])
 
+  const handleLanguageChange = useCallback(
+    (lng: string) => {
```

Comment:
this is sort of silly but maybe we could name this `lang` or `language`? we use `lng` pretty extensively to mean `longitude` elsewhere :)

Reply:
yeah, I was thinking the same thing, but i18n uses `lng` so I stuck with it. I'll change it up.